source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -92,30 +92,36 @@ RasterYuv420Semiplanar: class extends RasterImage {
 		}
 		result
 	}
-	resizeInto: func (target: This) {
+	resizeInto: func (target: This, sourceBox := IntBox2D new()) {
+		if (sourceBox hasZeroArea)
+			sourceBox = IntBox2D new(this size)
```

Comment:
I think it is evaluated only once. It does not seem to be a problem now, but maybe someone would like to change the `size` at some point.
Should I change it anyway ?